magic-square/src/magic_square/puzzle.clj

```diff
@@ -1,8 +1,26 @@
 (ns magic-square.puzzle)
 
-(def values [1.0 1.5 2.0 2.5 3.0 3.5 4.0 4.5 5.0])
-
 (defn magic-square [values]
   [[1.0 1.5 2.0]
    [2.5 3.0 3.5]
-   [4.0 4.5 5.0]])
+   [4.0 4.5 5.0]]
+   (def sumvalue (reduce + values))
+   (def magicno (/ sumvalue 3))
+   (def me (int (/ (count values) 2)))
+   (let [
+         mm (get values me)
+         mf (get values (+ me 3))
+         mb (get values (- me 3))
```

Comment:
It is not clear to me what this algorithm is doing.
You have to arrange the 9 numbers into a matrix, not choose new numbers. 
have you tried other numbers that can be arranged in the same order?